gotthard/cli.py

```diff
@@ -307,7 +305,11 @@ def get_spilo_stacks(cf, spilo_cluster):
                     'UPDATE_ROLLBACK_IN_PROGRESS', 'UPDATE_ROLLBACK_FAILED',
                     'UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS', 'UPDATE_ROLLBACK_COMPLETE']
 
-    for stack in cf.list_stacks(StackStatusFilter=stack_status)['StackSummaries']:
+    paginator = cf.get_paginator('list_stacks')
+    stacks = paginator.paginate(StackStatusFilter=stack_status,
+                                PaginationConfig = {'MaxItems': 10000}).build_full_result()['StackSummaries']
```

Comment:
As far as I understood, MaxItems gives you the number of items you get on a single run, before you need to request the NextToken, so 10K is set to avoid calling this function multiple times.